lib/validation.js

```diff
@@ -19,10 +19,6 @@
  * @module selectValidation
  */
 
-module.exports = function selectValidation(input, settings) {
-  if (input.target.value === '' && !settings.target.empty) {
-    return `${input.target.name} cannot be left blank!`;
-  }
-
+module.exports = function selectValidation() {
```

Comment:
@Snugug This is a very basic input plugin with very basic javascript. Originally created to test javascript injection. No stories currently exist to expand on this functionality - when they do we should def grow this plugin.